pkgs/misc/ptouch-print/default.nix

```diff
@@ -1,33 +1,49 @@
-{ lib, stdenv
+{ lib
+, stdenv
 , fetchgit
-, autoreconfHook
+, cmake
+, gettext
 , gd
+, git
 , libusb1
+, pkg-config
 }:
 
-stdenv.mkDerivation rec {
+stdenv.mkDerivation {
```

Comment:
why remove rec? Imo this makes no sense if the idea is to update it away from a specific git hash and instead use regular version numbers again eventually.